apps/web/svelte.config.js

```diff
@@ -20,9 +25,7 @@ const config = {
   ]),
 
   kit: {
-    adapter: adapter({
-      precompress: true
-    })
+    adapter: multiAdapter([netlifyAdapter(), nodeAdapter()])
```

Comment:
Also maybe it'd be worth choosing what adapter to use based on some environment variable, instead of always building both? Since I'm guessing building both would increase build times by quite a lot.

We could just do something like `adapter: process.env.BUILD_ENV == "netlify" ? netlifyAdapter() : nodeAdapter()` (and then add an environment variable BUILD_ENV with value "netlify" to the netlify config)

Reply:
Oh yeah that works as well